dataduct/s3/utils.py

```diff
@@ -119,6 +119,46 @@ def upload_dir_to_s3(s3_path, local_path, filter_function=None):
         local_path(file_path): Input path of the file to be uploaded
         filter_function(function): Function to filter out directories
     """
+    # 5MB
+    CHUNK_SIZE = 5242880
+
+
+    def multipart_upload(key_string, local_file_path):
+        import glob
+        import subprocess
+        import time
+        username = os.path.basename(os.path.expanduser('~'))
+        directory = '/tmp/multipart_upload_{}'.format(username)
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+        prefix = os.path.join(directory, 'tmp_upload')
```

Comment:
does this mean we cannot upload more than 1 file at a time? you may wish to generate a UUID for this upload.